crates/iota-aws-orchestrator/src/protocol/mod.rs

```diff
@@ -101,6 +101,28 @@ pub trait ProtocolMetrics {
             .collect()
     }
 
+    /// The command to retrieve the metrics from the nodes.
+    fn nodes_flamegraph_command<I, T>(
```

Comment:
we actually created a script that lets you run a command on all the validators. This avoid screating a separate orchestrator command for every operation we might want to perform. It's `crates/iota-aws-orchestrator/assets/run_command.sh`